File: src/main/java/com/example/schoolview/MainActivity.java
package com.example.schoolview;

import androidx.appcompat.app.AppCompatActivity;
import android.widget.Button;
import android.widget.RatingBar;

import android.os.Bundle;

public class MainActivity extends AppCompatActivity {



    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        final RatingBar ratingRatingBar = (RatingBar) findViewById(R.id.ratingBar);
        Button submitButton = (Button) findViewById(R.id. button);
        //final TextView ratingDisplayTextView = (TextView) findViewById(R.id.rating_display_text_View);

        //submitButton.setOnClickListener(new View.OnClickListener() {
//            @Override
//            public void onClick(View v) {
//                ratingDisplayTextView.setText("Your rating is: " + ratingRatingBar.getRating());


    }
}

File: src/main/java/com/example/mobileappassignment1/MainActivity.kt
package com.example.mobileappassignment1

import android.content.Intent
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Toast


class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val button = findViewById<Button>(R.id.button);

        button.setOnClickListener {

            Toast.makeText(this, "Thank You!", Toast.LENGTH_SHORT).show()

        }

        val button2 = findViewById<Button>(R.id.button2);

        button2.setOnClickListener {

            val intent = Intent(this, SecondaryActivity::class.java )

            startActivity(intent)

        }

    }
}
